tests-semanticdb/src/test/resources/metac.expect_2.12

```diff
@@ -1662,7 +1654,7 @@ Synthetics:
   withFilter => scala/collection/TraversableLike#withFilter().
   b => local5
   flatMap => scala/collection/generic/FilterMonadic#flatMap().
-  check$ifrefutable$1 => local6
+  local6 => local6
```

Comment:
while here it was "used", the rest of them ("local7", "local8", 9, 10. etc) are all `localX => localX`, so this is consistent.